src/webclients/components/sdfv/utils/attributes_table.ts

```diff
@@ -404,15 +411,19 @@ export function attrTablePutDict(
         const rowbox = $('<div>', {
             'class': 'container-fluid',
         }).appendTo(modal.body);
-        Object.keys(val).forEach(k => {
+        Object.keys(val).forEach(async k => {
             let v = val[k];
-            const attrProp = attributeTablePutEntry(
-                k, v, valMeta, val, elem, xform, rowbox, true, false, true
+            const row = $('<div>', {
+                class: 'row attr-table-row',
+            }).appendTo(rowbox);
+            const attrProp = await attributeTablePutEntry(
+                k, v, valMeta, val, elem, xform, row, true, false, true
             );
 
             if (attrProp.deleteBtn)
                 attrProp.deleteBtn.on('click', () => {
                     attrProp.keyProp?.getInput().val('');
+                    attrProp.keyProp?.markDeleted();
```

Comment:
why/when would I want to erase a property?

Reply:
This is used for dictionaries and data descriptors. If a data descriptor is to be removed, the corresponding property is deleted. The terminology may be slightly misleading (unintentionally, this crept in) - a property here is not equal to an SDFG property. It is more general here, so even elements of a dictionary (or dict property) are called properties.

Reply:
Needs a brief comment somewhere